python/feathub/feature_views/transforms/sliding_window_transform.py

```diff
@@ -39,8 +39,13 @@ def __init__(
         :param expr: A FeatHub expression composed of UDF and feature names.
         :param agg_func: The aggregation function or the name of the aggregation
                          function as string such as "MAX", "AVG".
-        :param window_size: The size of the sliding window.
+        :param window_size: The size of the sliding window. If it is zero, it means that
+                            the sliding window has infinite size. All the rows are
+                            aggregated.
         :param step_size: The step_size specifies how often the sliding windows starts.
+                          If the window size is zero, the step size must be zero, which
```

Comment:
Should we support window_size = 0 and step_size > 0 in the future? If yes, maybe it is better not to specify this short-term limitation in the constructor. We can throw an exception in the constructor if the given set of the parameters can not be supported yet.

Do we support window_size > 0 and step_size = 0? If no, would it be useful to explicitly throw an exception in the constructor?

Reply:
We should support window_size = 0 and step_size > 0 in the future. I updated the docs to avoid specifying the short-term limitation. And throw an exception in the constructor until it is supported in the future. 

And I don't think we should support window_size > 0 and step_size = 0 sliding window.